lib/auth_browser.dart

```diff
@@ -100,6 +100,10 @@ class BrowserOAuth2Flow {
   /// already logged in) and might get asked to grant the application access
   /// (if the application hasn't been granted access before).
   ///
+  /// If [force] is `true` the user will be forced to login and asked to grant
```

Comment:
From offline: what if `immediate` and `force` are both true? Which one "wins"?

Reply:
This is not defined for `runHybridFlow`:
https://github.com/dart-lang/googleapis_auth/blob/5bb42ae16d43c8c701e709125a3729bdfab61f03/lib/auth_browser.dart#L163-L181

Which also exposes both `force` and `immediate`. Have anyone found documentation on this?